files/contaodemo/theme/src/scss/components/_slider.scss

```
@@ -1,116 +1,129 @@
 $slider-breakpoint: lg;
-.slider {
+
+.swiper {
   $parent: &;
 
-  &-control {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    gap: 1rem;
+  --swiper-pagination-bottom: 0;
 
-    margin-block-start: 1rem;
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  gap: 1rem;
 
-    height: 3rem;
+  #{$parent}-wrapper {
+    flex-basis: 100%;
+  }
 
-    // fullsize on screens < 1024
-    .header-image & {
-      @include breakpoint(lg, max) {
-        padding-inline: $s-grid-outer-gutter--large;
-      }
-    }
+  #{$parent}-pagination {
+    position: inherit;
 
-    #{$parent}-menu,
-    a {
-      position: static;
-      top: 0;
-    }
+    // Reset default Swiper styles
+    order: 2;
+    width: auto;
+    line-height: 0;
```

```suggestion
    margin-top: unset;
    line-height: 0;
```

That should still exist

---

`margin-top` musste an dieser Stelle gar nicht resettet werden. War ein Fehler von mir im Ursprungscommit.